pdfarranger/pdfarranger.py

```diff
@@ -798,39 +798,41 @@ def iv_dnd_received_data(self, iconview, context, x, y,
             if ref_to:
```

Comment:
Is this if clause still useful?

Reply:
I tested to remove it and also on line 823 and dragged some pages and I dont see any problem with it. So maybe it does not need to be there..